Makefile

```diff
@@ -120,6 +130,26 @@ DOCKER_ARCH_TAG=$(ZARCH)
 
 FULL_VERSION:=$(ROOTFS_VERSION)-$(HV)-$(ZARCH)
 
+
+ifeq ($(PREEMPT_RT),1)
+  ifneq ($(ZARCH), amd64)
+    $(error "PREEMPT_RT is not supported for $(ZARCH)")
+  endif
+    KERNEL_FLAVOR=rt
+endif
+
+KERNEL_VERSION=$(KERNEL_VERSION_$(KERNEL_FLAVOR))
+KERNEL_COMMIT=$(KERNEL_COMMIT_$(ZARCH)_$(KERNEL_FLAVOR))
+
+# this is the branch we use for kernel builds. Can be overwritten from a command line
+KERNEL_BRANCH ?= eve-kernel-$(ZARCH)-$(KERNEL_VERSION)-$(KERNEL_FLAVOR)
+# this can be set to 'latest' from the command line to use development kernel
+KERNEL_DOCKER_TAG ?= $(KERNEL_BRANCH)-$(KERNEL_COMMIT)-$(KERNEL_COMPILER)
+# one can override the whole tag from the command line and set it to
+# output of make -f Makefile.eve docker-tag-${KERNEL_COMPILER} in github.com/lf-edge/eve-kernel
+#KERNEL_TAG ?= docker.io/lfedge/eve-kernel:$(KERNEL_DOCKER_TAG)
+KERNEL_TAG ?= docker.io/yashzededa/sbom:$(KERNEL_DOCKER_TAG)
```

Comment:
I think docker calls this a name; definitely not a tag.

Reply:
I agree with @eriknordmark , but if you look on all variables defined by the `tools/parse-pkgs.sh` they will also contain the name of the container + the tag, although they are called ???_TAG.... I think that's because of linuxkit, since `linuxkit pkg show-tag` shows the name of the container + its tag.... so IMO it's fine to let KERNEL_TAG....

Reply:
These are tags. Anything after the `:` in a container image name is a tag. What the contents of that tag are (as long as legitimate) is fair play and is part of the tag. `lkt pkg show-tag` uses the git tree hash, but to make a tag.

Reply:
@deitch My comment was about KERNEL_TAG - not about KERNEL_DOCKER_TAG. Latter is a tag after the colon.
But since parse-pkgs has been calling the whole name+tag a tag forever, this is not the time and place to change that.

Reply:
Ah, I had misunderstood. Thank you for correcting.

Yeah, the whole "is tag the whole thing or just the stuff after the `:`" is a painful nomenclature problem that has been around forever, and not just in eve. But this is not the place to fix it, agreed.